l3kernel/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ this project uses date-based 'snapshot' version identifiers.
 
 ## [Unreleased]
 
+### Added
+- Documentation for `\sys_if_timer_exsit_p:` and `\sys_if_timer_exsit:(TF)`.
```

Comment:
typo: *exsit* should be `\sys_if_timer_exist_p:` `\sys_if_timer_exist:(TF)`

Reply:
Should be OK now. Also corrected in commit message. Thanks! 😄